components/shared/workCard/helper.ts

```diff
@@ -1,19 +1,22 @@
 import {
   GeneralMaterialTypeCodeEnum,
+  ManifestationDetailsFragment,
   SearchWithPaginationQuery,
+  WorkFullWorkPageFragment,
   WorkTeaserSearchPageFragment,
 } from "@/lib/graphql/generated/fbi/graphql"
 
 export const displayCreators = (
-  creators: SearchWithPaginationQuery["search"]["works"][0]["creators"],
+  creators: WorkFullWorkPageFragment["creators"] | ManifestationDetailsFragment["contributors"],
```

Comment:
I'm suggesting not having the function to be specific to the types WorkFullWorkPageFragment["creators"] and  ManifestationDetailsFragment["contributors"]. Could it expect an array instead here?

```suggestion
  creators: string[],
```

Reply:
That's not specific enough for this case. Then TS wouldn't be happy with us giving it an array of objects. I think it's pretty nice for a specific use case :)